src/common/intrusive_lru.h

```diff
@@ -166,6 +166,14 @@ class intrusive_lru {
     evict();
   }
 
+  template <class F>
+  void for_each(F&& f) {
+    for (auto& v : lru_set) {
+      access(v);
```

Comment:
The previous version missed assuring that the `lru` isn't `nullptr`. Fixed.

Reply:
Why does this need a WITH_SEASTAR ifdef?

Reply:
The idea was to emphasize it doesn't pose a threat to the classical world, so -- despite affecting the `common` -- the big QA testing is not a must.

Reply:
Removed.